packages/DatePicker/src/components/Calendar/Calendar.styles.js

```diff
@@ -0,0 +1,141 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+import CalendarBaseStyle from "./CalendarBase.styles";
+
+function getNumber(token) {
+  return token.slice(0, -2) / 1;
+}
+
+const CalendarStyled = styled.div`
+  ${CalendarBaseStyle}
+
+  * {
+    box-sizing: border-box;
+    font-family: ${tokens.fontFamily.default};
+  }
+
+  .DayPicker_transitionContainer__horizontal {
+    transition: none;
+  }
+
+  .DayPicker_weekHeader {
+    top: 46px;
+
+    small {
+      ${stylers.fontSize(-1)};
+    }
+  }
+
+  .CalendarMonth_caption {
+    background-color: ${tokens.color.blackLighten80};
+    height: ${getNumber(tokens.space) * 4 + getNumber(tokens.space)}px;
```

Comment:
Why not use `parseInt()`? 
Better yet, why not use `stylers.spacer(5)`?

Reply:
I try to make the calculation more clear, but looks like should be ok just use `spacer(5)`, will remove `getNumber`!